packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ContestsList/ContestsList.tsx

```diff
@@ -74,12 +70,9 @@ const ContestsList = ({
   return (
     <>
       <div className="ContestsList">
-        {isAdmin &&
-        ((weightedTopicsEnabled ? !hasWeightedTopic : !stakeEnabled) ||
-          !isContestAvailable) ? (
+        {isAdmin && (!hasWeightedTopic || !isContestAvailable) ? (
           <EmptyContestsList
             hasWeightedTopic={hasWeightedTopic}
-            isStakeEnabled={stakeEnabled}
```

Comment:
not sure, but shouldn't we keep the `stakeEnabled` prop?

Reply:
no bc it is not important anymore, now we check if there is weighted topic rather than stake enabled. Thanks for checking in!